src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/services/NomisService.kt

```diff
@@ -20,26 +20,24 @@ class NomisService(
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
-  private fun getOffendersIds(offset: Long = 0, size: Long = 10) = prisonApiWebClient.get()
-    .uri("/api/offenders/ids")
-    .header("Page-Offset", offset.toString())
-    .header("Page-Limit", size.toString())
+  private fun getOffendersIds(page: Int = 0, size: Int = 10) = prisonApiWebClient.get()
```

Comment:
we were passing in the page size here instead of the offset by mistake, so hopefully this will fix the issue